src/freenet/client/async/SplitFileFetcherStorage.java

```diff
@@ -1640,7 +1640,12 @@ public void maybeClearCooldown() {
         fetcher.clearCooldown();
     }
 
+    /** Returns -1 if the request is finished, otherwise the wakeup time. */
     public long getCooldownWakeupTime(long now) {
+        // It is OK to use the main lock here.
+        // Cooldown is a separate lock because updating it is tricky.
+        // FIXME maybe set cooldown when finish and avoid this.
+        if(hasFinished()) return -1;
```

Comment:
Space between `if` and `(` to meet the [style guide](https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s4.6.2-horizontal-whitespace).